docs/orchestrating-elastic-stack-applications/elasticsearch/jvm-heap-dumps.asciidoc

```diff
@@ -20,15 +20,15 @@ To take a heap dump before the JVM process runs out of memory you can execute th
 [source,sh,subs="attributes,+macros"]
 ----
 kubectl exec $POD_NAME -- su elasticsearch -g root -c \
-  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep java)'
+  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep -n java)'
 ----
 
 If the Elasticsearch container is running with a random user ID, as for example on OpenShift, there is no need to substitute the user identity:
 
 [source,sh,subs="attributes,+macros"]
 ----
 kubectl exec $POD_NAME -- bash -c \
-  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep java)'
+  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep -n java)'
```

Comment:
```suggestion
  '/usr/share/elasticsearch/jdk/bin/jmap -dump:format=b,file=data/heap.hprof $(pgrep -f org.elasticsearch.bootstrap.Elasticsearch)'
```

Reply:
Assuming we won't change the Java class name in the future, LGTM!

Reply:
This may also be an issue when the process cmdline is too long. `pgrep -f org.elasticsearch.bootstrap.Elasticsearch` doesn't work on serverless, where `cat /proc/66/cmdline | wc` returns `2105`.

Reply:
What do you suggest instead @thbkrkr ? The original proposal from @jeanfabrice ?